content/2019-05-05_Sprint-na-vychode.md

```diff
@@ -0,0 +1,25 @@
+Title: Sprint na východě
+Date: 2019-05-05 00:00:00
+Author: Karolina Surma
+
+## Sprint? To jako běháte?
+
+Členové a členky české pythonní komunity se obvykle dvakrát do roka sejdou v nějakém koutu republiky. Letos jsem se podruhé vydala na společné řešení komunitních otázek, webových aplikací a ostatních aktivit kolem propagace programování v Pythonu. Jak je známo, začátečníci mají obvykle řadu obav před zapojením se mezi zkušené programátory a programátorky. Jak je taky známo, Pythonní komunita je inkluzivní a pořádá řadu aktivit mířených přímo na nováčky. Mírou úspěchu může být tedy nadpoloviční účast brněnských PyLadies (v hojném počtu 4) vedle zkušenějších kolegů z Moravy a Slezska. 
```

Comment:
Sjednoť pythonní a Pythonní, nebo klidně použij v obou případech jen Python komunita, to asi taky neva. Pokud přídavné jméno, pak s malým p